### DISCUSSION TITLE
Why does the n9e quick view select 10 hosts by default?

### DISCUSSION THREAD

**Discussion by wushengwusheng** [ccfos/nightingale#1280] [Q&A] [answered]
前端版本：5.13.0
后端版本：v5.13.0-445d03e0969fa519aa5ceb0ecc2b46e6231db05f

现象：在进入快捷视图界面，默认会选中10个主机。如需选择其他主机监控，需要先点击清理按钮，再进行选择；如果未进行清理，直接通过筛选其他机器，显示的属性还是之前选中的10台主机的监控项。

<img width="491" alt="image" src="https://user-images.githubusercontent.com/30250875/203248044-ebf916ea-43de-4450-9531-fe31289f2986.png">
<img width="1234" alt="image" src="https://user-images.githubusercontent.com/30250875/203248485-7be8a9e1-04cb-4705-96b3-373d8693998f.png">

**Comment by jsers** [comment]
现在默认会选中前 10 个选项
你搜索完再选中，还是基于之前选中的 10 个基础上的
如果只想选中个别指定的，需要先清空已选项

wushengwusheng ***@***.***> 于2022年11月22日周二 15:11写道：

**Reply by wushengwusheng** [reply]
这样做的目的是什么呀

**Comment by jsers** [comment, marked as answer]
> 现在默认会选中前 10 个选项
这个比如查询实例，对于查询体验和性能来说默认不选和全选都不如默认选中前 10 个选项

另外两点是比较正常的交互方式

wushengwusheng ***@***.***> 于2022年11月22日周二 18:40写道：

**Reply by wushengwusheng** [reply]
好的 是正常的就行～
